automated_archive/aa.py

```diff
@@ -22,11 +22,13 @@ def _clean_file(filepath, log):
     :param filepath: Path to ARCHIVE_DB.pl
     :return: Python dictionary keyed by original story id
     """
-    h = HTMLParser()
-    archive_db = codecs.open(filepath, "r", encoding="utf-8").read()
+    encoding = input('Encoding for the ARCHIVE_DB.pl file (default: "utf-8"): ')
```

Comment:
Might be worth mentioning the exact format for a couple of common encodings, just so the next user doesn't have to look it up and then typo it:
```suggestion
    encoding = input('Encoding for the ARCHIVE_DB.pl file, e.g. "utf-8", "latin_1", "cp1252" (default: "utf-8"): ')
```
("utf-8" is listed as "utf_8" at https://docs.python.org/3.8/library/codecs.html#standard-encodings so I'm not sure why we had a hyphen instead of an underscore in the original line, or whether that matters)

Reply:
I just took your suggestion (though had to do another commit thanks to ruff 😅) I don't think the hyphen vs underscore matters, I've always defaulted to the hyphen with no issues